p2p/peering.go

```diff
@@ -1678,3 +1723,44 @@ func (rp *RemotePeer) SendHeadersSent() bool {
 	rp.requestedHeadersMu.Unlock()
 	return sent
 }
+
+// GetInitState attempts to get initial state by sending a GetInitState message.
+func (rp *RemotePeer) GetInitState(ctx context.Context) (*wire.MsgInitState, error) {
+	const opf = "remotepeer(%v).GetInitState"
+	msg := wire.NewMsgGetInitState()
+	msg.AddTypes(wire.InitStateTSpends)
```

Comment:
let's either move these type constants into the method parameters, so it can be used to query anything (we might want to fetch other mempool txs using this, too), or take the *wire.MsgGetInitState as a parameter directly.